library/src/com/astuetz/PagerSlidingTabStrip.java

```diff
@@ -194,8 +189,11 @@ public PagerSlidingTabStrip(Context context, AttributeSet attrs, int defStyle) {
         isPaddingMiddle = a.getBoolean(R.styleable.PagerSlidingTabStrip_pstsPaddingMiddle, isPaddingMiddle);
         tabTypefaceStyle = a.getInt(R.styleable.PagerSlidingTabStrip_pstsTextStyle, Typeface.BOLD);
         tabTypefaceSelectedStyle = a.getInt(R.styleable.PagerSlidingTabStrip_pstsTextSelectedStyle, Typeface.BOLD);
-        tabTextAlpha = a.getFloat(R.styleable.PagerSlidingTabStrip_pstsTextAlpha, HALF_TRANSP);
-        tabTextSelectedAlpha = a.getFloat(R.styleable.PagerSlidingTabStrip_pstsTextSelectedAlpha, OPAQUE);
+        tabTextColorSelected = a.getColorStateList(R.styleable.PagerSlidingTabStrip_pstsTextColorSelected);
+        if (tabTextColorSelected == null) {
+            tabTextColorSelected = tabTextColor;
+        }
+
```

Comment:
IMHO, I think many of these customisation attributes are creating problems because each one adds complexity.

So if you've tested this and are happy it doesn't break anything, it's up to you - I'll open a new issue for discussion on what I think is too complex about the lib, and you can decide if it's something you want to tackle or not.